res/main/i18n/fr/statuses.json

```diff
@@ -1,6 +1,6 @@
 {
 	"jack": {
-		"label": "JACK:",
+		"label": "JACK :",
```

Comment:
This space looks suspicious. Isn't it an occasional key hit?

Reply:
Nope, it's correct.
It's a French language rule: Spaces before (most) punctuations.